addons/sourcemod/scripting/ttt/ttt_taser.sp

```diff
@@ -64,46 +67,50 @@ public Plugin myinfo =
 public void OnPluginStart()
 {
 	TTT_IsGameCSGO();
-	
+
 	LoadTranslations("ttt.phrases");
-	
+
 	BuildPath(Path_SM, g_sConfigFile, sizeof(g_sConfigFile), "configs/ttt/config.cfg");
 	Config_Setup("TTT", g_sConfigFile);
 	g_iCreditsTaserHurtTraitor = Config_LoadInt("ttt_hurt_traitor_with_taser", 2000, "The amount of credits an innocent or detective will recieve for discovering a traitor with their zues/taser.");
-	
+	g_bAddSteamIDtoLogs = Config_LoadBool("ttt_steamid_add_to_logs", true, "Should we add steam id to all log actions? Prevent abusing with namefakers");
+	g_iSteamIDLogFormat = Config_LoadInt("ttt_steamid_log_format", 1, "Which steam id format to you prefer? 1 - SteamID2 (STEAM_1:1:40828751), 2 - SteamID3 ([U:1:81657503]) or 3 - SteamID64/CommunityID (76561198041923231)");
```

Comment:
Being dumb, shouldn't this go `1 - SteamID64`, `2 - SteamID2`, `3 - SteamID3` so that the 2 & 3 line up.

Reply:
I have no idea why it's marking those lines as "new". The only things I changed are related to making the OnTased forward. If you look in the current ttt_taser.sp in the main branch, it's the exact same.

Reply:
@Bobakanoosh Oh yeah... so it is.  Ignore my comment then! 😕